lib/jobs/install-os.js

```diff
@@ -62,28 +61,14 @@ function installOsJobFactory(
     /**
      * @memberOf InstallOsJob
      *
-     * Validate the input options.
+     * Validate the input options, only implement those validation that task-schema cannot covers
      */
     InstallOsJob.prototype._validateOptions = function() {
         assert.string(this.context.target);
-        assert.string(this.options.completionUri);
-        assert.string(this.options.profile);
-
-        // Some of the install task (such as coreos) still hard coded the repo in
-        // the profile/kickstart file, So we cannot assert repo&version here
-        //
-        // TODO: If all install tasks use the same options format,
-        // then uncomment following  lines:
-        // assert.string(this.options.repo);
-        // assert.string(this.options.version);
-        // assert.string(this.options.rootPassword);
-        // assert.string(this.options.hostname);
-        // assert.string(this.options.domain);
 
         if (this.options.networkDevices) {
             _.forEach(this.options.networkDevices, function(dev) {
                 assert.string(dev.device);
-                var vlanIds;
                 if (dev.ipv4) {
                     assert.isIP(dev.ipv4.ipAddr, 4);
```

Comment:
Yes, ajv only has 'ipv4' format support. I keep the `ipv4` here is to align with the code about `ipv6`.